src/validation.h

```diff
@@ -917,6 +917,7 @@ class ChainstateManager
     void Reset();
 };
 
+/** DEPRECATED! Please use node.chainman instead. May only be used in validation.cpp internally */
```

Comment:
In commit "validation: Mark g_chainman DEPRECATED" (fa24ea2e67d9812a7b57f9820996a8695a8fef37)

Could enforce this by deleting extern declaration from validation.h, and just declaring it an extern variable right where it's used in init.cpp.

Reply:
I might remove it in a later pull altogether, so I opted for the smallest diff for now